cmd/podman/containers/run.go

```diff
@@ -227,7 +227,7 @@ func run(cmd *cobra.Command, args []string) error {
 	if runRmi {
 		_, rmErrors := registry.ImageEngine().Remove(registry.GetContext(), []string{imageName}, entities.ImageRemoveOptions{})
 		if len(rmErrors) > 0 {
-			logrus.Errorf("%s", errorhandling.JoinErrors(rmErrors))
+			logrus.Warn(errorhandling.JoinErrors(rmErrors))
```

Comment:
Interesting suggestion. How do you envision this situation happening? All I can think of is two `run --rmi` containers started in perfect sync, or `podman rmi` run at juuuuust the right moment when a `run --rmi` container finishes but before cleanup. Is there a realistic case where this can happen?

Reply:
Could just be paranoia, but I think that errors like this are probably the cause of most of our flakes, so I don't think it would hurt to check

Reply:
But errors that say "Could not remove XYZ, because XYZ does not exists" feels stupid to me.

Reply:
```console
$ /bin/rm foo
/bin/rm: cannot remove 'foo': No such file or directory
```
That said, I agree, and the version I pushed an hour ago filters out `ErrImageUnknown`. Since this is a big stretch for me, and since there's not really any possible way to test this condition, everyone please take a special-close look.